ntoskrnl/ke/bug.c

```diff
@@ -273,7 +273,6 @@ KeRosDumpStackFrames(IN PULONG_PTR Frame OPTIONAL,
     {
         /* Get the current frames (skip the two. One for the dumper, one for the caller) */
         RealFrameCount = RtlCaptureStackBackTrace(2, FrameCount, (PVOID*)Frames, NULL);
-        DPRINT1("RealFrameCount =%lu\n", RealFrameCount);
```

Comment:
Please revert.

Reply:
If truly useless, have a separate PR for that.